src/preload.js

```diff
@@ -91,9 +91,16 @@ const elements = {
       window.location.href.includes("/mix/")
     ) {
       if (currentPlayStatus === statuses.playing) {
-        const row = window.document.querySelector(this.playing_title).closest(this.tracklist_row);
-        if (row) {
-          return row.querySelector(this.album_name_cell).textContent;
+        const playing_title = window.document.querySelector(this.playing_title);
+        
+        if (playing_title) {
+          const row = playing_title.closest(this.tracklist_row);
+          if (row) {
+            const album_name_cell = row.querySelector(this.album_name_cell);
+            if (album_name_cell) {
+              return album_name_cell.textContent;
+            }
+          }
```

Comment:
This seems like unnecessary overhead for a function that mostly already works fine the occasional glitch out when going on another album happens but it is so minor since when you switch to another album you either anyways will start playing it or immediately switch back to another playlist which will fix the album name with the next song anyways.

Reply:
It was occasionally causing the mute to fail, but I rearranged the other bit some so maybe i can revert that change